packages/debug/browser.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env browser */
+
+const debugCookie =
+  (document.cookie.match(/(^|;)\\s*hops_debug\\s*=\\s*([^;]+)/) || []).pop() ||
```

Comment:
```suggestion
  (document.cookie.match(/(^|;)\s*hops_debug\s*=\s*([^;]+)/) || []).pop() ||
```

No need escape backslashes anymore.